freud/order.pyx

```diff
@@ -611,6 +611,14 @@ cdef class Steinhardt(_PairCompute):
             &self.thisptr.getQl(),
             freud.util.arr_type_t.FLOAT)
 
+    @_Compute._computed_property
+    def particle_harmonics(self):
+        """:math:`\\left(N_{particles}, 2*l+1\\right)` :class:`numpy.ndarray`:
+        The raw array of \\overline{q}_{lm}(i)."""
```

Comment:
This docstring needs to indicate the indexing convention. Is `m=0` at the beginning or the center?